hpcgap/lib/mapping.gi

```diff
@@ -195,9 +202,15 @@ InstallGlobalFunction( Image, function ( arg )
 
         return [];
 
+      else
+        ErrorNoReturn( "families of <elm> or <coll> and Source(<map>) ",
+                       "don't match, ",
+                       "maybe <elm> or <coll> are not contained in ",
+                       "Source(<map>) or aren't homogeneous lists," );
       fi;
     fi;
-    Error( "usage: Image(<map>), Image(<map>,<elm>), Image(<map>,<coll>)" );
+    ErrorNoReturn( "usage: Image(<map>), Image(<map>,<elm>), ",
+                   "Image(<map>,<coll>)" );
```

Comment:
However, in my opinion, it is fine to use `<coll>` here. Here you're only giving a name for a potential argument, and in doing so suggesting that it could be a collection. You're not implying that a variable called `coll` already exists.

Reply:
Yupp. I'll keep it that way.